setup.sh

```diff
@@ -62,46 +62,46 @@ if [[ -z $QUIET_INTRO ]]; then
     # GITHUB_TOKEN=01234567890abcdef01234567890abcdef012345 git adsk
     #
     set +e
-    STORED_GITHUB_ENTERPRISE_ACCOUNT=$(git config --global adsk.github.account)
-    STORED_GITHUB_ENTERPRISE_SERVER=$(git config --global adsk.github.server)
+    STORED_GITHUB_ENTERPRISE_ACCOUNT=$(git config --global $KIT_ID.github.account)
+    STORED_GITHUB_ENTERPRISE_SERVER=$(git config --global $KIT_ID.github.server)
     set -e
 
     if [[ -z $GITHUB_TOKEN ]]; then
         if [[ -z $STORED_GITHUB_ENTERPRISE_ACCOUNT ]]; then
-            echo -n 'Please enter your Autodesk username and press [ENTER]: '
+            echo -n 'Please enter your GitHub Enterprise username and press [ENTER]: '
         else
-            echo -n "Please enter your Autodesk username and press [ENTER] (empty for \"$STORED_GITHUB_ENTERPRISE_ACCOUNT\"): "
+            echo -n "Please enter your GitHub Enterprise username and press [ENTER] (empty for \"$STORED_GITHUB_ENTERPRISE_ACCOUNT\"): "
```

Comment:
Some of our engineers confuse GitHub Enterprise with github.com. Therefore I used the company name here. Maybe we should make this an enterprise.config setting?